Added analogy intro & extra questions (#2)
lnbook/lnbook: 01-what-is.asciidoc

[[whatis_chapter]]
== What is the Lightning Network?

Think of a thunderstorm, where electricity is ionizing channels of air to dissipate excess energy and neutralize voltage differences that have been building up during the storm. 
A channel of ionized air forms when the voltage or potential difference is greater than the breakdown voltage (..kV) between the different regions in the cloud. 
As the electric current makes its way through the fractal network of ionized air channels it chooses the path of least resistance to reach its destination, a path is formed between the charged up cloud and the ground of earth below. 
Then a bright flash of light followed by a loud bang occurs as the current dissipated in the path of ionized air channels creates a hot plasma.
The Lightning Network is named after this natural phenomenon, where the voltage differences are channel balances and the current is a stream of bitcoin of which ownership is transmitted through those channels.
When lightning strikes the ground, the potential differences are settled at the cost of dissipating energy and making a loud bang.

> In the Lightning Network this happens when payment channels close, the final state of the channel is transmitted to the Bitcoin network with a fee for the miners to include it in the next block.

Reviewer: I learned physics in my native language, therefore I'm having a really difficult time to follow the analogy, due to not being familiar with the terminology. I admit it's a badass analogy, but making readers stop reading from the very first paragraph may not be the best way to go.  
The counter argument count be is that, at this point everyone is paying full attention and whoever chews through this first paragraph will find the following parts easier.  
Anyhow, not against it, but I think it worth a discussion.

Author: I will make a nice visual diagram to go with it, so that the concept is clear even without having learned the physics jargon.

Reviewer 2: Agree with Adam, you risk to lose 90% of readers at the first paragraph :) I have a PhD in physics but still I'm not sure what the point was.

Reviewer 3: I like the analogy, but agree with some of the comments here that its too complex and risks losing readers straight away.   It may be better to start with language such as "lightning is a way to improve upon the bitcoin protocol with additional functionality that increases transaction throughput and reduces settlement time. "

== Why use the Lightning Network?

> A second layer protocol can magnify the amount of transactions possible on the settlement layer, so that the base layer will not be overloaded with transactions as Bitcoin gains more active (transacting) participants. 

Reviewer: > as Bitcoin gains more active (transacting) participants. 

I think it's uneccessarily complex phrasing.

Author: Yeah, I tend to use very technical terms. What would you suggest here?
“As Bitcoin adoption grows in the world” perhaps.

Reviewer: ACK

Instead small value payments can be instantly transmitted in a Lightning Channel, or a chain of channels through the Lightning Network.

> As these channels are not settled on-chain with every transaction, it will become more difficult to differentiate the smaller payments that occured on the channel from eachother.

Reviewer: typo

```suggestion
As these channels are not settled on-chain with every transaction, it will become more difficult to differentiate the smaller payments that occured on the channel from each other.
```

Reviewer: Also I'm not entirely sure it's correct.

With this move to a higher protocol layer, payments also gain more privacy and fungibility depending on the chosen route.

== How to use the Lightning Network?

Many interoperable implementations are under active development using the BOLT specifications. 

> It can be run as a Mobile Wallet App on any smartphone, a desktop light-client or routing-node.

Reviewer: Why the capitalization?

Author: Lightning Network was capitalized in the title, BOLT is an abreviation. Mobile wallet app should be lowercase indeed!

Reviewer: Yes, I didn't argue with the LN capitalization.

```suggestion
It can be run as a mobile application, a desktop light-client or routing-node.
```
